### ISSUE TITLE
issue getting to cloned directory

### ISSUE THREAD

**Issue by billyadm123** [learn-co-curriculum/dsc-data-science-env-installation-v2-1#1]
Receiving the following error message:  Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by lauravlad** [comment]
I received the same message. Were you able to find a solution?

**Comment by billyadm123** [comment]
No. But I think eugina sent me a solution. I can send it to you.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by lauravlad** [comment]
yes, please.
Can you send it to me?

**Comment by billyadm123** [comment]
1)Generate SSH key using ssh-keygen -t rsa -b 4096 -C "your_email@foo.com".   2)Copy the output of cat ~/.ssh/id_rsa.pub 3)Paste the above copied outpur   into your Github profile -> Settings -> SSH and GPG Keys -> Add new   SSH key.
--
try this one. It helped me to solve this problem

This is what I was sent.  I haven't tried it yet but I hope it helps

**Comment by hoffm386** [comment]
The quick fix is to change the link you are cloning to the HTTPS version, which does not require as complicated of setup.   So, the bash command to run would be:
```
git clone https://github.com/learn-co-curriculum/dsc-data-science-env-installation-v2-1.git
```

Complete setup instructions for SSH are [here](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh)
